makefile.shade

```diff
@@ -508,7 +508,7 @@ var RC_FILES = '${FindAllFiles("Resource.rc", BOOTSTRAPPER_EXE_NAME, BOOTSTRAPPE
             var runtimeAssemblies = new HashSet<string>();
             foreach (var project in runtimeProjects)
             {
-                if (project == "dotnet.core45.managed" && target.Framework == "aspnet50")
+                if (project == BOOTSTRAPPER_CORECLR_MANAGED_NAME && target.Framework == "aspnet50")
```

Comment:
so far we use the string in make file directly. no a good practice but I don't intent to tackle this issue in this PR so as to keep the fix coherent. #1244 is just filed to track this issue.